portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,81 @@
 
 package com.google.sps.servlets;
 
+import java.util.*; 
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns "Hello Jay Saleh!". TODO: modify this file to handle comments data */
+/** Servlet that writes an list of messages as a response. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
+  /** Creates a "comment" entity and stores in database */
```

Comment:
nit: I would re-write this as: Creates and stores a {@code Comment} in the database.

@code tag will make it obvious this is a object/class.

Reply:
Removed override comments